package.json

```diff
@@ -33,34 +33,34 @@
   "scripts": {
     "lint": "eslint .",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap -J -R specy --no-coverage test/*test.js",
+    "test:unit": "tap --disable-coverage test/*test.js",
```

Comment:
`specy` isn't available in tap 18, so I just removed it and went with the default reporter.

Reply:
It's alright, but please move `disable-coverage` and files to the `.taprc` file

```
disable-coverage: true
files:
  - test/**/*.test.js
```

Reply:
Updated.